.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
 	"editor.codeActionsOnSave": {
-		"source.fixAll.eslint": true,
+		"source.fixAll.eslint": "always"
 	},
```

Comment:
I note this bit was probably accidentally added in #37418 (they also tried adding `.vscode/settings.json` to `.gitignore`, redundantly). Probably it should have been in the user's config instead, leaving this to only specify exclusions and locations of linter programs.

But I'm going to punt on all this to @tbradsha who has strong opinions on this file. 😀

Reply:
> strong opinions on this file

Indeed, mostly centering around this PR's statement that "it's annoying"; I'll put up a post today soliciting more feedback. In the meantime, it's probably best to just remove the `editor.codeActionsOnSave` section.